ansible/files/telemetry.sh

```diff
@@ -85,6 +85,9 @@ echo "export PYSPARK_SUBMIT_ARGS=\"--packages com.databricks:spark-csv_2.10:1.2.
 
 source $HOME/.bashrc
 
+# Setup Jupyter notebook
+aws s3 cp $TELEMETRY_CONF_BUCKET/bootstrap/jupyter_notebook_config.py ~/.jupyter/jupyter_notebook_config.py
```

Comment:
ansible-vault can be a pain as well to use when multiple devs are involved. What about storing the credentials on S3 and applying them to a template of `jupyter_notebook_config.py`?